superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -322,7 +324,7 @@ export default function transformProps(
       axisLabel: { formatter },
       scale: truncateYAxis,
       name: yAxisTitle,
-      nameGap: yAxisTitleMargin,
+      nameGap: convertNumber(yAxisTitleMargin),
```

Comment:
I think this change needs to be applied in Boxplot and Mixed Timeseries charts as well? They probably have the same bug

Reply:
I fixed the issues in Boxplot and Mixed Timeseries charts